app/views.py

```diff
@@ -30,14 +30,14 @@ def isLoggedIn():
     return True
 
 
-def getUser():
+def getUser(username):
     if not isLoggedIn():
         return {'error': "Not logged in."}
 
     ts = pymesync.TimeSync(baseurl=app.config['TIMESYNC_URL'],
                            test=app.config['TESTING'],
                            token=session['token'])
-    user = ts.get_users(username=session['username'])
+    user = ts.get_users(username=username)
 
     return user
```

Comment:
@subnomo You should return an empty dictionary so that other parts of the code can just go

```
if not user:
    # ...
```

to check if the user object gotten from get_user() is valid